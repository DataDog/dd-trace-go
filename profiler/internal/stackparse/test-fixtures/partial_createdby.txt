goroutine 1 [running]:
main.foo()
	/example/main.go:182 +0x123
created by main.main

goroutine 2 [running]:
main.foo()
	/example/main.go:183 +0x123
created by main.foo
