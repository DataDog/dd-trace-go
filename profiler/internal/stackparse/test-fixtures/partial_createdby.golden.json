{
  "Errors": [
    "invalid file:line ref on line 5: \"\"",
    "invalid file:line ref on line 10: \"\""
  ],
  "Goroutines": []
}
